tools/congestion-model/run_eval.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# exit on error
+set -euo pipefail
+
+# show each executed command
+# set -x
+
+NUM_SHARDS=6
+WARMUP=500
+ROUNDS=1500
+TX_POOL_SIZE=200
+
+OPTIONS_STRING="--rounds ${ROUNDS} --warmup ${WARMUP} --shards ${NUM_SHARDS} --tx-pool-size ${TX_POOL_SIZE}"
+
+echo "# Delay VS Utilization"
+cargo run -q --release -- --workload RelayedHot --strategy SmoothTrafficLight ${OPTIONS_STRING} 
```

Comment:
Why --release?

Reply:
Because debug builds are a lot slower in execution.